media-fonts/iosevka/iosevka-1.14.1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit font
+
+DESCRIPTION="Slender typeface for code, from code"
+HOMEPAGE="http://be5invis.github.io/Iosevka"
+SRC_URI="https://github.com/be5invis/${PN}/releases/download/v${PV}/01-${P}.zip
+https://github.com/be5invis/${PN}/releases/download/v${PV}/02-${PN}-term-${PV}.zip"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sh ~sparc ~x86 ~x86-fbsd ~x64-macos"
```

Comment:
Where did you copy KEYWORDS from? The choice is pretty odd. Preferably please restrict it to arches you've tested personally (+ `~x86` if you've tested on `~amd64`).